### PR TITLE
Added static hosting Fixes #8

### LINKED ISSUE #8: Add static file hosting
To host the client-side library.

### PR DESCRIPTION
**Pull request by TheThirdOne**

### CODE REVIEW

File: app/handler.go [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+	"os"
+	"path/filepath"
+	
+	"github.com/gorilla/mux"
+	"github.com/hackedu/maestro/router"
+)
+
+var (
+	// StaticDir is the directory containing static assets.
+	StaticDir = filepath.Join(defaultBase("github.com/hackedu/maestro/app"), "static")
```
Comment (zachlatta) [inline app/handler.go]: What happens if `GOPATH` is not set? Say on a production machine where we just deploy a static binary?

**Comment by TheThirdOne** [comment]
I honestly have no clue.

File: app/handler.go (hunk 1 above)
Comment (zachlatta) [inline app/handler.go]: Assuming your comment was meant for here:

> I honestly have no clue.

Not super familiar with it, but https://github.com/GeertJohan/go.rice may be helpful.

**Comment by zachlatta** [comment]
Gonna just go ahead and merge for now. Features > code quality/edge cases right now.

**Pull request merged**